contrib/packaging/bootupctl-update.service

```
@@ -0,0 +1,13 @@
+[Unit]
+Description=Update Bootloader on boot
+Documentation=https://github.com/coreos/bootupd
+ConditionPathExists=/dev/disk/by-label/EFI-SYSTEM
```

```suggestion
ConditionPathExists=/dev/disk/by-partlabel/EFI-SYSTEM
```

As that's what's used in https://github.com/coreos/bootupd/blob/main/src/efi.rs#L39